.eslintrc.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
nice one :) you can stay with this config, but if you wish to have super nice and verified one, check airbnb linter settings ;) not in this branch